src/app/helpers/utils/utils.ts

```diff
@@ -0,0 +1,9 @@
+export function toSnakeCase (value: string) {
+  return value.replace(/([A-Z])/g, letter => `_${letter.toLowerCase()}`);
+};
+
+export function keysToSnakeCase (obj: Object) {
+  let res: Object = {};
```

Comment:
the res variable doesn't need a typing, she gets one implicit when it's assigned.